docs/yymnist/README.md

```diff
@@ -0,0 +1,80 @@
+README
+==========
+
+# Introduction
+
+This document is intended to walk a user through the necessary steps to run object detection on the [yymnsit](https://github.com/cmu-sei/yymnist) dataset with Juneberry.
+
+# Setup 
+Install opencv, Juneberry, and pull down the yymnsit repo
+<br>
+`git config --global http.proxy http://cloudproxy.sei.cmu.edu:80/`
```

Comment:
This should not be in the public repo. They know nothing about our proxy, nor should they.